harvard-lts/mirador-help-plugin
# Adding help component.

**Adding the help component*
* * *


# What does this Pull Request do?
This adds the help plugin component.

# How should this be tested?

A description of what steps someone could take to:
* npm run start
* go to http://localhost:3000
* confirm the help modal window exists on the viewer.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No 
- integration tests? No
